### ISSUE TITLE
WebUI expanding sections by default

### ISSUE THREAD

**Issue by Index154** [BananaPuncher714/LethalQuantities#85]
I have a minor gripe with the WebUI expanding all the settings that have been modified / enabled whenever a preset is loaded. Could you change it so they are collapsed instead by default?

The way it is currently makes it somewhat annoying to tweak anything because it can take a lot of scrolling to reach specific settings. Example image:
![image](https://github.com/BananaPuncher714/LethalQuantities/assets/105119758/87c1452c-12f9-47a8-966d-7aabfddbf19b)

I think it would be fine if it started out like this instead with the major sections being expanded but the individual entries within being hidden:
![image](https://github.com/BananaPuncher714/LethalQuantities/assets/105119758/68639e59-69de-4b4c-9f87-6e0379e7a2bc)

**Comment by BananaPuncher714** [comment]
Yes, I'm planning on adding a preferences section that will allow you to specify different settings, including whether or not sections are hidden by default.
